torch/csrc/jit/passes/onnx.cpp

```diff
@@ -427,7 +427,12 @@ void NodeToONNX(
     WithInsertPoint insert_point_guard(new_block);
     WithCurrentScope scope_guard(*g, n->scope());
     py::object raw_output = onnx.attr("_run_symbolic_function")(
-        g, new_block, n, py_inputs, env, operator_export_type);
+        g->shared_from_this(),
```

Comment:
Helpful to add a note to alert future readers?

Reply:
added comment and doxygen warning for `owningGraph`